### ISSUE TITLE
Showcase

### ISSUE THREAD

**Issue by flaviolivolsi** [theatre-js/website#8]
What do you think about implementing a showcase section in the website? From what I understand there are already some pretty big cases to show (see Gucci), and I would love to see what folks are doing with Theatre.js!

**Comment by tomByrer** [comment]
Yes, many folks need to see a library in use before being convinced to use it themselves.

For short term, a small section on the home page with a few select projects/websites that use TheatreJS.

Long term: a separate page that lists all the projects that use TheatreJS, any open repos, related blog posts, etc.  Should have a JSON file as source, so it can be re-used for 'Awesome TheatreJS', etc.
